extensions/amp-story-interactive/0.1/amp-story-interactive-abstract.js

```diff
@@ -788,4 +810,14 @@ export class AmpStoryInteractive extends AMP.BaseElement {
     };
     this.storeService_.dispatch(Action.ADD_INTERACTIVE_REACT, update);
   }
+
+  /**
+   * Toggles the tabbable elements (buttons, links, etc) to only reach them when page is active.
+   * @param {boolean} toggle
+   */
+  toggleTabbableElements_(toggle) {
```

Comment:
The entire page has some accessibility attributes that should ensure that none of its content is tabbable. This method should be a no-op, and if it's not we should file a ticket and solve this problem at a high level.

Reply:
When I tab on desktop, it does focus the tabbable elements if I don't fix it from here:
![image](https://user-images.githubusercontent.com/22420856/103236046-ac5d4f00-4911-11eb-8e53-3535b1f36528.png)

Checking now, links on other pages also get focused! This is definitely a higher-level problem but quizzes working inside a shadow-root might not be influenced by whatever logic we write on the story pages to disable tabbing through other pages. Ticket is https://github.com/ampproject/amphtml/issues/31747 for the general problem, but as I said, we might need to implement sth element-wise if the amp-story generalization doesn't apply to shadow-root elements...

Reply:
I wonder if we want to take out `toggleTabbableElements_` and save it for the work in #31747.
Or we could make a TODO to remove it when  #31747 is fixed.

Reply:
Added the `TODO`